pyproject.toml

```diff
@@ -45,6 +45,7 @@ Issues = "https://github.com/secure-systems-lab/securesystemslib/issues"
 [project.optional-dependencies]
 crypto = ["cryptography>=37.0.0"]
 gcpkms = ["google-cloud-kms", "cryptography>=37.0.0"]
+azurekms = ["azure-identity", "azure-keyvault-keys"]
```

Comment:
this should  have `"cryptography>=37.0.0"` in there too I think

Reply:
(the reason cryptography is optional in securesystemslib even though you can do almost nothing without it, is potential pip integration: verifying ed25519 is possible without cryptography, and cryptography is C dependency which pip dislikes)

Reply:
Updated, resolving..